papers.html

```diff
@@ -136,6 +136,26 @@
+      <p></p>
+      <h5>Xing et al. 2024</h5>
+      <p></p>
+
+      <p>
+        <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#Xing20241029" aria-expanded="false" aria-controls="Xing20241029">
+          Details
+        </button>
+      </p>
+        <div class="collapse" id="Xing20241029">
+          <div class="card card-body">
+            <p><b>Authors: </b> Zepei Xing, Vicky Kalogera, Tassos Fragos, Jeff J. Andrews, Simone S. Bavera, Max Briel, Seth Gossage, Konstantinos Kovlakas, Matthias U. Kruckow, Kyle A. Rocha, Meng Sun, Philipp M. Srivastava, Emmanouil Zapartas</p>
+            <p><b>Access: </b> <a href="https://arxiv.org/abs/2410.20415" target="_blank">arXiv:2410.20415</a> || <a href="https://ui.adsabs.harvard.edu/abs/2024arXiv241020415X/abstract" target="_blank">NASA ADS</a></p>
+            <p><b>Abstract: </b>The existence of a mass gap of $3-5\,M_{\odot}$ between the heaviest neutron stars (NSs) and the lightest black holes (BHs), inferred from the BH mass distribution in low mass X-ray binaries (LMXBs), has been suggested for decades. The recently reported gravitational-wave source GW230529 has been confidently identified as a neutron star--black hole (NSBH) merger, with the BH mass falling within this lower mass gap. This detection provides strong evidence against the existence of the latter and introduces new implications for the coalescing NSBH population, including a revised BH mass distribution and an updated local merger rate. In this study, we employ \texttt{POSYDON}, a binary population synthesis code that integrates detailed single- and binary-star models, to investigate coalescing NSBH binaries formed through isolated binary evolution. In particular, we focus on the BH mass distribution of the intrinsic NSBH merger population. We find that, with a high common-envelope efficiency of $\alpha_{\rm{CE}} =2 $, the BH masses in NSBH mergers concentrate in the lower mass gap, aligning more closely with observations. However, after accounting for the constraints of the selection bias against mass-gap BHs in LMXBs, which suggests that the maximum NS birth mass is below $\simeq 2\,M_{\odot}$, we find that introducing a high $\alpha_{\rm{CE}}$ is not required to match observations. Additionally, we explore the impact of core-collapse supernova kicks on the BH mass distribution and the local merger rate density of NSBH mergers. Finally, we present the property distributions of observable NSBH mergers from our simulation and find that they match well with the observations. With a self-consistent estimate of BH spins, we find that the fraction of electromagnetic counterparts in observable populations is $\approx 4-30\%$, depending on different NS equations of state. Future detections of coalescing NSBH binaries would provide invaluable insights into SN mechanisms, common envelope evolution, and NS physics. </p>
```

Comment:
Please convert the LaTeX to plain text or make use of html functionalities.

Reply:
```suggestion
            <p><b>Abstract: </b>The existence of a mass gap of 3-5&nbsp;<i>M</i><sub>&odot;</sub> between the heaviest neutron stars (NSs) and the lightest black holes (BHs), inferred from the BH mass distribution in low mass X-ray binaries (LMXBs), has been suggested for decades. The recently reported gravitational-wave source GW230529 has been confidently identified as a neutron star--black hole (NSBH) merger, with the BH mass falling within this lower mass gap. This detection provides strong evidence against the existence of the latter and introduces new implications for the coalescing NSBH population, including a revised BH mass distribution and an updated local merger rate. In this study, we employ POSYDON, a binary population synthesis code that integrates detailed single- and binary-star models, to investigate coalescing NSBH binaries formed through isolated binary evolution. In particular, we focus on the BH mass distribution of the intrinsic NSBH merger population. We find that, with a high common-envelope efficiency of <i>&alpha;</i><sub>CE</sub> =2, the BH masses in NSBH mergers concentrate in the lower mass gap, aligning more closely with observations. However, after accounting for the constraints of the selection bias against mass-gap BHs in LMXBs, which suggests that the maximum NS birth mass is below &simeq; 2&nbsp;<i>M</i><sub>&odot;</sub>, we find that introducing a high  <i>&alpha;</i><sub>CE</sub> is not required to match observations. Additionally, we explore the impact of core-collapse supernova kicks on the BH mass distribution and the local merger rate density of NSBH mergers. Finally, we present the property distributions of observable NSBH mergers from our simulation and find that they match well with the observations. With a self-consistent estimate of BH spins, we find that the fraction of electromagnetic counterparts in observable populations is &approx; 4-30%, depending on different NS equations of state. Future detections of coalescing NSBH binaries would provide invaluable insights into SN mechanisms, common envelope evolution, and NS physics. </p>
```

Reply:
thanks, updated